fe/fe-core/src/main/java/org/apache/doris/transaction/DatabaseTransactionMgr.java

```diff
@@ -742,6 +742,37 @@ public void replayBatchRemoveTransaction(List<Long> txnIds) {
         }
     }
 
+    public void replayBatchRemoveTransaction(BatchRemoveTransactionsOperationV2 operation) {
+        writeLock();
+        try {
+            int numOfClearedTransaction = 0;
+            if (operation.getLatestTxnIdForShort() != -1) {
+                while (!finalStatusTransactionStateDequeShort.isEmpty()) {
+                    TransactionState transactionState = finalStatusTransactionStateDequeShort.pop();
```

Comment:
In old version,  when execute ```replayBatchRemoveTransaction```, txnId could be removed when txnId locates in head of deque.
So in V2, how to ensure the consistency between latestTxnIdForShort/latestTxnIdForLong and finalStatusTransactionStateDequeShort/finalStatusTransactionStateDequeLong.
Is there the case that latestTxnIdForShort not in finalStatusTransactionStateDequeShort?
How about add a check here?

Reply:
only when is there a bug, never encounter the unconsistent case here